src/components/inputs/InputAmount/types.ts

```diff
@@ -22,4 +22,5 @@ export interface InputAmountProps {
   showUnit?: boolean;
   readOnly?: boolean;
   showMaxButton?: boolean;
+  customCoinIcon?: React.ReactElement;
```

Comment:
I had some doubt if it was better to add an optional `icon` prop to TokenItem instead but let's keep it like that.

Reply:
Yes i was thinking about the same, but this is easier for now